source/fundamentals/crud/write-operations/modify.txt

```diff
@@ -13,11 +13,18 @@ Modify Documents
 Overview
 --------
 
-In this guide, you can learn how to modify documents in a MongoDB
-collection using two different kinds of operations: 
+In this guide, you can learn how to use the {+driver-long+} to modify
+documents in a MongoDB collection by performing the following operations:
 
-- :ref:`Update <csharp-update-operation>`
-- :ref:`Replace <csharp-replace-operation>`
+- :ref:`Update Operations <csharp-update-operation>`
+- :ref:`Replace Operations <csharp-replace-operation>`
+
+The {+driver-short+} provides the following methods to change documents,
```

Comment:
Suggestion:

I think "modify documents" is more suitable than "change documents" for the following reasons:
- "modify documents" was used in the previous paragraph.
- When "change" is read as an adjective instead of a verb, the term "change documents" can potentially be confused with "change stream documents".